Kudu.Core/Functions/FunctionManager.cs

```diff
@@ -296,17 +296,13 @@ private string GetFunctionScriptPath(string functionName, JObject functionInfo)
                     }
                 }
 
-                if (string.IsNullOrEmpty(functionPrimary))
-                {
-                    // TODO: should this be an error?
-                    return functionPath;
-                }
                 return functionPrimary;
             }
         }
 
         private Uri FilePathToVfsUri(string filePath, bool isDirectory = false)
         {
+            if (string.IsNullOrEmpty(filePath)) return null;
```

Comment:
is there a reason not not fix the caller instead (never passing null)?  callee should throw if null or empty etc.   easier to find bug that way.